src/cli.js

```diff
@@ -19,6 +20,8 @@ const cli = meow([`
     -c, --config  - Operate from config file. In this mode only token, verbose, and
                     debug flags apply.
 
+    --full        - Returns the full tree of contributions rather than the default
```

Comment:
The default vs full isn't explicated anywhere that I can see. Can you add that?

Reply:
Yeah, I think the documentation is a bit lacking given the changes. I'll update it.

Reply:
Thanks.